halyard-config/src/main/java/com/netflix/spinnaker/halyard/errors/v1/HalconfigException.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.halyard.errors.v1;
+
+import lombok.Data;
+
+/**
+ * This is the base exception class that needs to be thrown by all validators.
+ */
+@Data
```

Comment:
I still think @Data here is extraneous - a @Getter on HalconfigAlert seems more appropriate